src/routes/overviews/bufnmf.svx

```diff
@@ -0,0 +1,134 @@
+
+### Bases
+
+The “bases” can be thought of as spectral templates (or filter curves) that BufNMF discovers in the course of analyzing the spectrogram. As described above, BufNMF analyzes a spectrogram to determine which parts of the spectrum always (or often) occur together in time. These parts of the spectrum that occur together are then combined into one spectral template (or “basis”). You can see below the bases for the drum analysis conducted above. Each plot is a spectrum (the _x_ axis is spectral bins, representing frequency, and the _y_ axis is magnitude, representing loudness).
+
+<Image
+src="/overviews/bufnmf/01_Bases_plot.png"
+label="(blue = kick, orange = snare, green = hi-hat)"
+/>
+
+Basis 1 only has one peak of energy very low in the spectrum representing the kick drum. Basis 2 has the various partials of the snare drum head’s resonant frequency. Basis 3 has lots of energy higher in the spectrum, representing the hi-hat sound. (See NMFFilter for how these bases can be used to filter a sound signal through these spectra.)
+
+### Activations
+
+Each base is also paired with an “activation” which represents how present each of these bases is throughout the audio buffer (through time). Below you can see the activations from the drum analysis conducted above overlaid on the waveforms of their corresponding components. For each plot the _x_ axis is time and the _y_ axis is loudness.
+
+<Image
+src="/overviews/bufnmf/00_resynth_buffer_with_activations.png"
+label="(blue = kick, orange = snare, green = hi-hat)"
+/>
+
+As you can see, each of these looks like an envelope follower for the drum instrument component they represent. These activations could be used to control the loudness of a signal through time. Here is an example of each of these activations modulating the loudness of pink noise.
+
+<Audio
+src="/overviews/bufnmf/04_activation_0_enveloping.mp3"
+label="Activation of Component #1 enveloping pink noise."
+/>
+
+<Audio
+src="/overviews/bufnmf/04_activation_1_enveloping.mp3"
+label="Activation of Component #2 enveloping pink noise."
+/>
+
+<Audio
+src="/overviews/bufnmf/04_activation_2_enveloping.mp3"
+label="Activation of Component #3 enveloping pink noise."
+/>
+
+When BufNMF decomposes the mono drum buffer into the three component buffers (as seen and heard above) it does this by combining each component’s basis with just that component’s activation to resynthesize the component’s resulting audio buffer. For example BufNMF combines just the “hi-hat basis” (i.e., spectral template) with just the “hi-hat activation” (loudness envelope through time) to resynthesize the resulting “hi-hat component” audio buffer.
+
+There are a suite of objects in the FluCoMa toolkit that can be used along with BufNMF’s bases and activations. Check out [NMFFilter](/reference/nmffilter), [NMFMatch](/reference/nmfmatch), [NMFMorph](/reference/nmfmorph), and [BufNMFCross](/reference/bufnmfcross).
```

Comment:
If there were any other resources you wanted to link I've been using a <ResourceLink /> component under a `## Related Resources` header at the bottom. You can see the example rendered here: https://learn.flucoma.org/reference/umap